### PR TITLE
Add lots of docs, mostly best practices.

### PR DESCRIPTION
**Pull request by brian-brazil**
Start out on the visualisation docs.

Expand overview, remove federation as a feature we don't have that yet.

Switch example variable to follow naming scheme.

### CODE REVIEW

File: content/docs/practices/alerting.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+### Metamonitoring
+
+It's important to have confidence that your monitoring is working, accordingly
+you should have alerts that there's sufficient Prometheuses, Alertmanagers,
+PushGateways and other monitoring infrastructure running to alert on any
+problems.
+
+As always, if you can alert on symptoms rather than causes this helps reduce
+noise. For example a blackbox test that alerts are getting from PushGateway to
```
Comment (brian-brazil) [inline content/docs/practices/alerting.md]: We're referring to our particular Alertmanager and PushGateway implementations, thus I think they're proper nouns and get capitals.

**Comment by brian-brazil** [comment]
I think that's all of your comments covered.

Not using contractions feels very unnatural to me.

**Comment by juliusv** [comment]
@brian-brazil Thanks! Re-reading and then reading the second half now.

I agree in some cases about the contractions. @opensourcegrrrl (our doc writer) has always told me to avoid them, but I'm not sure if we have to be so strict about it.

**Comment by juliusv** [comment]
As discussed on IRC, I'm going to merge this as-is now and send a followup PR with some changes later. Easier for everyone :)

**Comment by opensourcegrrrl** [comment]
"Expand abbreviations, such as e.g., because they add complexity for non-native readers, are used inconsistently, and are unnecessary." http://infobits.eu/tech-doc/style-guide/

**Comment by opensourcegrrrl** [comment]
In an instance such as, "When doesn't it fit?", the emphasis should be on _not_, yet that part is contracted. "When does it not fit?" might be the clearer option.

**Pull request merged**